includes/documents/class-wcpdf-invoice.php

```diff
@@ -196,7 +196,7 @@ public function init_settings() {
 			array(
 				'type'			=> 'setting',
 				'id'			=> 'disable_for_statuses',
-				'title'			=> __( 'Disable for:', 'woocommerce-pdf-invoices-packing-slips' ),
+				'title'			=> __( 'Disable for', 'woocommerce-pdf-invoices-packing-slips' ),
```

Comment:
same as with the other conversation: this will break translations, so let's keep the colon for now!